docs: describe ordering in http query results (#6024)
grafana/loki: docs/sources/api/_index.md

}
```

> The items in the `values` array are sorted by their timestamp in a descending (when using `direction=backward`) or ascending (when using `direction=forward`) order.

Reviewer: I'm suggesting a rewrite of this to try to be more clear about ordering. It should also make any future localization (translations to other languages) easier.

```suggestion
The items in the `values` array are sorted by timestamp.
The most recent item is first when using `direction=backward`.
The oldest item is first when using `direction=forward`.
```

Author: done!

    <label key-value pairs>
  },
  "values": [
    [
      <number: second unix epoch>,
      <string: value>
    ],
    ...
  ]
}
```

> The items in the `values` array are sorted by their timestamp in an ascending order.

Reviewer: ```suggestion
The items in the `values` array are sorted by timestamp, and the oldest item is first.
```

Author: done!

}
```

> The items in the `values` array are sorted by their timestamp in a descending (when using `direction=backward`) or ascending (when using `direction=forward`) order.

Reviewer: Suggesting same prose here as in the earlier commit suggestion.
```suggestion
The items in the `values` array are sorted by timestamp.
The most recent item is first when using `direction=backward`.
The oldest item is first when using `direction=forward`.
```

Author: done!
